_overviews/scala3-book/packaging-imports.md

```diff
@@ -93,90 +134,178 @@ A note before moving on:
 
 > Import clauses are not required for accessing members of the same package.
 
-
-
 ### Importing one or more members
 
 In Scala you can import one member from a package like this:
 
+{% tabs package_4 %}
+{% tab 'Scala 2 and 3' for=package_4 %}
+
 ```scala
 import scala.concurrent.Future
 ```
 
+{% endtab %}
+{% endtabs %}
+
 and multiple members like this:
 
+{% tabs package_5 %}
+{% tab 'Scala 2 and 3' for=package_5 %}
+
 ```scala
 import scala.concurrent.Future
 import scala.concurrent.Promise
 import scala.concurrent.blocking
 ```
 
+{% endtab %}
+{% endtabs %}
+
 When importing multiple members, you can import them more concisely like this:
 
+{% tabs package_6 %}
+{% tab 'Scala 2 and 3' for=package_6 %}
+
 ```scala
 import scala.concurrent.{Future, Promise, blocking}
 ```
 
+{% endtab %}
+{% endtabs %}
+
 When you want to import everything from the *scala.concurrent* package, use this syntax:
 
+{% tabs package_7 class=tabs-scala-version %}
+{% tab 'Scala 2' for=package_7 %}
+
+```scala
+import scala.concurrent._
+```
+
+{% endtab %}
+
+{% tab 'Scala 3' for=package_7 %}
+
 ```scala
 import scala.concurrent.*
 ```
 
+{% endtab %}
+{% endtabs %}
 
 ### Renaming members on import
 
 Sometimes it can help to rename entities when you import them to avoid name collisions.
 For instance, if you want to use the Scala `List` class and also the *java.util.List* class at the same time, you can rename the *java.util.List* class when you import it:
 
+{% tabs package_8 class=tabs-scala-version %}
+{% tab 'Scala 2' for=package_8 %}
+
+```scala
+import java.util.{List => JavaList}
+```
+
+{% endtab %}
+
+{% tab 'Scala 3' for=package_8 %}
+
 ```scala
 import java.util.{List as JavaList}
```

Comment:
just for idiomatic cleanup
```suggestion
import java.util.List as JavaList
```

Reply:
ok there's quite a lot of outdated syntax from the pre-3.0.0 days so I added a commit to clean this